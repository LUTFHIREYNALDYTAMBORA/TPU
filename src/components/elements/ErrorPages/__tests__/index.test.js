import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import ErrorPages from '../ErrorPages';

describe('src/components/elements/ErrorPages', () => {
  test('render correctly', () => {
    const shallow = new ShallowRenderer();
    const tree = shallow.render(<ErrorPages />);
    expect(tree).toMatchSnapshot();
  });
});
