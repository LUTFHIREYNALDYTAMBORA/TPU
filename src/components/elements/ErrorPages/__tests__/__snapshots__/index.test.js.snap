// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/elements/ErrorPages render correctly 1`] = `
<section
  className="m-auto min-h-screen flex items-center justify-center"
>
  <div
    className="text-center"
  >
    <h1
      className="text-9xl font-extrabold tracking-wider text-cyan-500"
    >
      
    </h1>
    <p
      className="text-2xl font-semibold uppercase py-4"
    >
      
    </p>
    <Link
      href="/"
    >
      <a
        className="bg-cyan-500 hover:bg-cyan-600 font-semibold rounded-lg text-red-50 p-2"
      >
        Home
      </a>
    </Link>
  </div>
</section>
`;
